mlrun/datastore/datastore_profile.py

```diff
@@ -311,12 +339,26 @@ class DatastoreProfileAzureBlob(DatastoreProfile):
     client_secret: typing.Optional[str] = None
     sas_token: typing.Optional[str] = None
     credential: typing.Optional[str] = None
+    bucket: typing.Optional[str] = None
+
+    @pydantic.validator("bucket")
+    def check_bucket(cls, v):
+        if not v:
+            warnings.warn(
+                "The 'bucket' attribute will be mandatory starting from version 1.9",
+                DeprecationWarning,
```

Comment:
It turns out that we use `FutureWarning` everywhere else in the code base, so we should probably change this (all instances in this file).
```suggestion
                FutureWarning,
```